source/CommonJobs/CommonJobs.Migrations/201301161535_CleanLastModifiedBy.cs

```diff
@@ -36,8 +36,12 @@ public override void Up()
 
             DocumentStore.DatabaseCommands.UpdateByIndex("Raven/DocumentsByEntityName",
                                                          new IndexQuery {  },
-                                                         patchs.ToArray(), 
-                                                         allowStale: true);
+                                                         patchs.ToArray(),
+                                                         //TODO: VALIDATE THIS
```

Comment:
The new new BulkOperationOptions class passed as a parameter. Again, a reminder since I didn't test migrations.